crates/ruma-client-api/CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ Breaking changes:
 - The `retry_after_ms` field of `ErrorKind::LimitExceeded` was renamed to
   `retry_after` and is now an `Option<RetryAfter>`, to add support for the
   Retry-After header, according to MSC4041 / Matrix 1.10 
+- `sync::sync_events::v3::Timeline::from_empty` now returns `true` when the
+  `limited` or `prev_batch` fields are set.
```

Comment:
```suggestion
- `sync::sync_events::v3::Timeline::is_empty` now returns `false` when the
  `limited` or `prev_batch` fields are set.
```

Also this is a bug fix, not a breaking change.

Reply:
Fixed the typo. I marked this is a breaking change because `Timeline::is_empty` is part of the public API. The documentation before the change just said

> Returns true if there are no timeline updates.

Which I thought was a little vague on what counts as a "timeline update". "no new events" is a reasonable interpretation, and so I was worried there might be some users that are relying on the old semantics.

I can switch it to a bug fix entry if you'd like.

Reply:
Yes, please mark as bug fix. The main reason that fn even exists is for the serde attribute.